DEHPEcosimPro/ViewModel/Dialogs/HubMappingConfigurationDialogViewModel.cs

```diff
@@ -180,9 +220,11 @@ public bool CanContinue
         /// <param name="dstController">The <see cref="IDstController"/></param>
         /// <param name="statusBar">The <see cref="IStatusBarControlViewModel"/></param>
         public HubMappingConfigurationDialogViewModel(IHubController hubController,
-            IDstController dstController, IStatusBarControlViewModel statusBar, IMappingConfigurationService mappingService) :
+            IDstController dstController, IStatusBarControlViewModel statusBar, IMappingConfigurationService mappingService, INavigationService navigation) :
```

Comment:
`mappingService` and `navigation` missing docs. Also `navigation` should be `navigationService`